lib/modules/dosomething/dosomething_reportback/includes/ReportbackItemTransformer.php

```diff
@@ -18,6 +18,7 @@ public function index($parameters) {
     }
     catch (Exception $error) {
       // @TODO: Potentially log error to watchdog.
+      http_response_code('404');
```

Comment:
Curious if this is similar code to #6163 -- where we want a 200 with empty array if no results found for an index endpoint. (e.g. `signups?campaigns=23`)